src/scss/03_molecules/form/_form-row.scss

```diff
@@ -34,21 +34,28 @@ $formRowAppendConfig: (
   &[data-children-count="2"] {
```

Comment:
Qui i numeri mi sembrano un po' modificati a caso 😄  Qual è la ratio dietro queste modifiche? Sono state concordate con tutto il team di designer dell'azienda?

Reply:
In realtà no, e si al momento sono decisi a priori sulla base dei nuovi sviluppi. La PR l'ho aperta proprio per iniziare a capire assieme come e cosa modificare